### DISCUSSION TITLE
Repeated Gisgus syntax

### DISCUSSION THREAD

**Discussion by saintmalik** [Comp-Labs/comp-labs-website#30] [Ideas]
Hello there, noticed you are implementing the gisqus comment repeatedly in your markdown file.

werent you able to implement it via ```src/theme/BlogPostItem/index.js``` ?

I feel its unhealthy to repeat those syntax

Regards

**Comment by saintmalik** [comment]
i think i got the reason, it was appearing site wide

**Comment by RudraSen2** [comment]
Yes, There was some issue after putting Giscus Component inside `src/theme/BlogPostItem/index.js`. The mapping was incorrect and other components were also not working. Anyway, think before you post!

**Reply by saintmalik** [reply]
no issue man, your blog is nice also
